bin/yaml/libraries.yaml

```diff
@@ -26,4 +26,587 @@ libraries:
         symlink: libs/llvm/{name}
         check_file: include/llvm/Config/llvm-config.h
         targets:
-          - trunk
\ No newline at end of file
+          - trunk
+      ulib:
+        type: github
+        method: nightlyclone
+        repo: stefanocasazza/ULib
+        check_file: configure
+        targets:
+          - trunk
+      benchmark:
+        type: github
+        method: nightlyclone
+        repo: google/benchmark
+        subdir: google-benchmark
+        check_file: README.md
+        targets:
+          - trunk
+      fmt:
+        type: github
+        method: nightlyclone
+        repo: fmtlib/fmt
+        check_file: CMakeLists.txt
+        targets:
+          - trunk
+      eigen:
+        type: gitlab
+        method: nightlyclone
+        repo: libeigen/eigen
+        check_file: README.md
+        targets:
+          - vtrunk
```

Comment:
is the `vtrunk` important here?

Reply:
Well, the code put that on front of trunk because of what function was used, so it was installed there, and now that's in the properties file as "libs.eigen.versions.trunk.path=/opt/compiler-explorer/libs/eigen/vtrunk"
It's not exactly consistent, but it's the way it is atm.